presto-docs/src/main/sphinx/release/release-0.214.rst

```diff
@@ -0,0 +1,51 @@
+=============
+Release 0.214
+=============
+
+General Changes
+---------------
+
+* Add feature and session toggles for *top-N row number* optimization.
+* Add ``ngrams`` function to generate the N-grams from an array.
+* Add ``qdigest`` type and associated functions.
+* Add statistics calculator support for ``TINYINT``, ``SMALLINT`` and ``VARBINARY`` literals.
+* Add system to delay query execution until a minimum number of workers nodes are available. If the requirement is not met within a time limit, the query is failed.
+* Improve error message for broken :func:``array_sort`` comparator.
+* Fix memory leak related to query tracking in coordinator. Completed queries are supposed to be purged shortly after its completion. However, a 0.213 regression broke this.
+* Lazily build hash table for ``MapBlock`` when the hash table is used in ``SingleMapBlock``.
```

Comment:
Should this be framed somehow in terms of how this will affect the administrator or user of Presto?

Reply:
Discussed offline with @raghavsethi. We decided to remove it.

Reply:
This is a substantial performance improvement that we definitely should mention. However, it needs to be phrased in terms of user impact, not implementation details. For example:
```
* Improve performance of queries that read maps but do not access any keys
  by building the hash table lazily.
```
(assuming the above description is accurate)